slides/kube/namespaces.md

```diff
@@ -1,236 +1,193 @@
-class: namespaces
-name: namespaces
+# Namespaces
 
-# Improving isolation with User Namespaces
+- We cannot have two resources with the same name
```

Comment:
Unsure if this is coming before or after the "rng, two ways" bit, but this might need to be clarified a bit, in light of that?